components/feature/webcompat/src/main/assets/extensions/webcompat/manifest.json

```diff
@@ -2,7 +2,7 @@
   "manifest_version": 2,
   "name": "Mozilla Android Components - Web Compatibility Interventions",
   "description": "Urgent post-release fixes for web compatibility.",
-  "version": "100.0.0",
+  "version": "100.2.0",
```

Comment:
Just to confirm, that we are jumping from one to two intentionally? :)

Reply:
Oh, sorry, I somehow missed the notification, @Amejia481.

Yeah, we landed `.1` in mozilla-central only as the changes were irrelevant for mobile, so this was on purpose. Thanks for checking! :)

Reply:
No problem, thanks for the clarification :)